api/router.mjs

```diff
@@ -0,0 +1,42 @@
+import { NOT_FOUND } from './defs.mjs'
+import * as db from './db.mjs'
+
+const handlers = {}
+const setter = (p) => (_, k, v) => (handlers[`${p}/${k}`] = v)
+const getter = (p) => (cache, k) => cache[k] || (cache[k] = ROUTE(`${p}/${k}`))
+const ROUTE = (p) => new Proxy({}, { get: getter(p), set: setter(p) })
+
+export const GET = ROUTE('GET/api')
+export const POST = ROUTE('POST/api')
```

Comment:
This code is a bit tricky, because it use `Proxy`, it is a way to have a nicer syntax to add routes:

```js
// instead of writting:
handlers['GET/link/discord'] = /* handler code */

// we can write
GET.link.discord = /* handler code */
```